### ISSUE TITLE
Why the custom dist proto

### ISSUE THREAD

**Issue by tsloughter** [oltarasenko/epmdless#13]
Is `epmdless_proto` really needed? Is there a history here that in earlier versions this was required but in 21+ it would work fine to only have the epmd module and use the standard Eralng dist proto?

**Comment by oltarasenko** [comment]
Hey, sorry for a bit of delay, @tsloughter I think that you're right. It's more like legacy stuff we used on previous versions of Erlang.

As I recall one of the things we were trying to fix is an active socket inside the Erlang distribution. So we have added the following lines with this purpose:
https://github.com/oltarasenko/epmdless/blob/master/src/epmdless_tls_dist_proxy.erl#L360-L376

(Our production was kind of killed by the message spikes, so we had to add these as a sort of backpressure. Not sure if it's relevant atm, would have to double-check the OTP code).
